example-jobs/SampleBuildFlow-example.groovy

```diff
@@ -1,15 +1,31 @@
 import jenkins.automation.builders.FlowJobBuilder
 
-def oahMaster= new FlowJobBuilder(
+def exampleFlow = new FlowJobBuilder(
         name: 'GeneratedFlowJob',
         description: 'this our first stab at it',
         jobs:['job1', 'job2']
 ).build(this);
 
-oahMaster.with{
+exampleFlow.with{
   logRotator{
       numToKeep(365)
   }
+}
+
+def exampleFlowWithWorkspace = new FlowJobBuilder(
+        name: 'flow-job-with-workspace',
+        description: 'a build flow job with its own workspace',
+        jobs:['job1', 'job2']
+).build(this)
+
+exampleFlowWithWorkspace.with{
+    scm {
+        git('https://github.com/cfpb/qu.git', 'master')
+    }
+
+    configure { node ->
+        node << buildNeedsWorkspace(true)
```

Comment:
if that is the case, we should make it a default. I was thinking of expanding our scmUtils. I can work it into it then.